src/CheckboxPage.js

```diff
@@ -0,0 +1,118 @@
+  constructor(props) {
+    super(props);
+    this.checkboxes = [];
+    this.initCheckbox = checkboxEl => this.checkboxes.push(new MDCCheckbox(checkboxEl));
+  }
+
+  componentWillUnmount() {
+    this.checkboxes.forEach(checkbox => checkbox.destroy());
+  }
+
+  render() {
+    return (
+      <div>
+        <div className='mdc-checkbox demo-checkbox' ref={this.initCheckbox}>
+          <input type='checkbox'
+                 className='mdc-checkbox__native-control'/>
+          <div className='mdc-checkbox__background'>
+            <svg className='mdc-checkbox__checkmark'
+                 viewBox='0 0 24 24'>
+              <path className='mdc-checkbox__checkmark-path'
+                    fill='none'
+                    stroke='white'
+                    d='M1.73,12.91 8.1,19.28 22.79,4.59'/>
+            </svg>
+            <div className='mdc-checkbox__mixedmark'></div>
+          </div>
+        </div>
+
+        <div className='mdc-checkbox demo-checkbox' ref={this.initCheckbox}>
+          <input type='checkbox'
+                 className='mdc-checkbox__native-control'/>
+          <div className='mdc-checkbox__background'>
+            <svg className='mdc-checkbox__checkmark'
+                 viewBox='0 0 24 24'>
+              <path className='mdc-checkbox__checkmark-path'
+                    fill='none'
+                    stroke='white'
+                    d='M1.73,12.91 8.1,19.28 22.79,4.59'/>
+            </svg>
+            <div className='mdc-checkbox__mixedmark'></div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+class CheckboxDemos extends Component {
+  constructor(props) {
+    super(props);
+    this.checkboxes = [];
+    this.initCheckbox = checkboxEl => this.checkboxes.push(new MDCCheckbox(checkboxEl));
+  }
+
+  componentWillUnmount() {
+    this.checkboxes.forEach(checkbox => checkbox.destroy());
+  }
+
+  renderCheckboxVariant(title, {checked, indeterminate}) {
+    return (
+      <div>
+        <h3 className='mdc-typography--subheading2'>{title}</h3>
+        <div className='mdc-checkbox demo-checkbox' ref={this.initCheckbox}>
+          <input type='checkbox'
+                 className='mdc-checkbox__native-control'
+                 defaultChecked={checked}
+                 ref={input => input.indeterminate = indeterminate}/>
+          <div className='mdc-checkbox__background'>
+            <svg className='mdc-checkbox__checkmark'
+                 viewBox='0 0 24 24'>
+              <path className='mdc-checkbox__checkmark-path'
+                    fill='none'
+                    stroke='white'
+                    d='M1.73,12.91 8.1,19.28 22.79,4.59'/>
+            </svg>
+            <div className='mdc-checkbox__mixedmark'></div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div>
+        {this.renderCheckboxVariant('Checked', {checked: true})}
+        {this.renderCheckboxVariant('Indeterminate', {indeterminate: true})}
```

Comment:
nit: intedeterminate is more a state than a variant.....but whatevs

(theres really only 1 variant of checkbox, which makes this page a little sparse)

Reply:
I think it's worth showing indeterminate, otherwise people might not even realize we have support for it.

Reply:
Keeping indeterminate